### PR TITLE
EnBW is not a power grid operator

### PR DESCRIPTION
**Pull request by sb12**
The EnBW power grid was split into Netze BW GmbH and TransnetBW GmbH in 1997. See also #5537

### CODE REVIEW

**Comment by sb12** [comment]
Not sure what to do about the duplicate id errors in the checks?
`Error - Duplicate id 'edis-fc3a87' in:`
`  E.DIS`
`  ./data/operators/power/substation.json`

**Comment by arch0345** [comment]
My bad, I forgot to tell you to add "e.dis" to the matchNames for E.DIS Netz GmbH in #5788. This should be fixed in [`47313fa`](https://github.com/osmlab/name-suggestion-index/commit/47313fa2b6186c653f9beb97dfc5a94a589cc912)

File: data/operators/power/line.json [hunk 1]
```diff
@@ -1653,33 +1641,6 @@
         "power": "line"
       }
     },
-    {
-      "displayName": "EnBW",
-      "id": "enbw-d5e323",
-      "locationSet": {"include": ["de"]},
-      "matchNames": [
-        "enbw ag",
```
Comment (arch0345) [inline data/operators/power/line.json:1661]: `enbw ag` should either be excluded or included in a matchNames of another entry

File: data/operators/power/line.json [hunk 2]
```diff
@@ -369,18 +369,6 @@
         "power": "line"
       }
     },
-    {
-      "displayName": "Badenwerk AG",
```
Comment (arch0345) [inline data/operators/power/line.json:373]: `badenwerk ag` and `badenwerk` should either be excluded or included in a matchNames of another entry

**Review by arch0345 — CHANGES_REQUESTED** [review]
A few changes are needed to ensure that these won't be added again whenever the latest planet data is collected.

File: data/operators/power/line.json (hunk 1 above)
Comment (sb12) [reply data/operators/power/line.json:1661]: I excluded them, as the new operator could be both either Netze BW or TransnetBW.

File: data/operators/power/line.json (hunk 2 above)
Comment (sb12) [reply data/operators/power/line.json:373]: I excluded them, as the new operator could be both either Netze BW or TransnetBW.
Badenwerk is the predecessor company of EnBW.

**Review by arch0345 — APPROVED** [review]

**Pull request merged**